scenarioo-server/src/test/java/org/scenarioo/rest/builds/BuildsResourceIntegrationTest.java

```diff
@@ -0,0 +1,47 @@
+package org.scenarioo.rest.builds;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource(locations = "classpath:test-application.properties")
+public class BuildsResourceIntegrationTest {
```

Comment:
@mi-we The problem is that this test probably indirectly sets a context path on ContextPathHolder (a ugly singleton we should maybe get rid of) and once this test has been run, the StepIdentifierTest now fails because of different context path being initialized there.